Makefile

```diff
@@ -135,6 +144,13 @@ else
 	sudo tar -C /usr/local/kubebuilder --strip-components=1 -zvxf envtest-bins.tar.gz
 endif
 
+.PHONY: golang
+golang: ## Ensure golang is installed
+ifeq ($(shell which go 2>/dev/null), )
+	curl -sSLo golang.tar.gz "https://go.dev/dl/go$(GOLANG_VERSION).$(GOLANG_ARCH).tar.gz"
+	tar -C $(GOLANG_INSTALL_PATH) -xzf golang.tar.gz
```

Comment:
We are not installing it in the default location (`/usr/local/go`) to avoid issues with lack of permissions in ci-operator.